### DISCUSSION TITLE
ontology site not available

### DISCUSSION THREAD

**Discussion by t-duan** [dracor-org/dracor-frontend#227] [Show and tell]
Hi there,
The ontology site (https://dracor.org/ontology) which is linked to https://dracor.org/doc/what-is-dracor/ seems to be unavailable.
Best regards,
t-duan

**Comment by t-duan** [comment]
https://dracor.org/ontology ist available, but not the link http://www.visualdataweb.de/webvowl/#iri=https://dracor.org/ontology on the "what-is-dracor"-site.

**Comment by lehkost** [comment]
Thanks for the hint, the initial WebVOWL service was shut down in the meantime. A fix was just provided through https://github.com/dracor-org/dracor-frontend/commit/d252f08f3c411cb7986a5d374fe2d1a18500ad9e, it will appear with the next site update.
